easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.x-GCC-8.2.0-2.31.1-hpcx.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '4.0.x'
+versionsuffix = '-hpcx'
+
+homepage = 'http://www.open-mpi.org/'
+description = """Mellanox flavored Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+sources = [{
+    'filename': 'openmpi-4.0.x.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/open-mpi',
+        'repo_name': 'ompi',
+        'commit': '03cf3e4',
+    },
+}]
+checksums = ['ef53605823b01499e09844a610614a555724603c72a1bb9a5ca59b35389f67ec']
```

Comment:
This checksum is never going to be the same due to how it is created.

Reply:
Why? Is there any way to fix that?

Reply:
github produces the tar file on demand by first making a checkout of that commit causing directories to have timestamps that differs every time.
Don't think there is a way to fix that.

@boegel ??

Reply:
@boegel could you please advise?

Reply:
Sorry for not coming back to this...

For now, there's no way to reliable checksum tarballs that were created via `git_config`, see also discussion in https://github.com/easybuilders/easybuild-framework/pull/2727